### ISSUE TITLE
Please improve the appearance. It looks a little bit old.

### ISSUE THREAD

**Issue by hiensumi** [cpeditor/cpeditor#1254]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is this feature provided in the master branch?

- [X] I have checked the changelog

### The problem you are facing

There are lots of ides with huge amounts of themes and custom appearance, but they do not have many features of CP editor.
I love CP editor but the appearance should be developed more.

### Describe the feature you'd like

May be add some new features to customize themes, cursor styles.
Change the GUI of the checker, logger...

### Are you willing to contribute to this?

- [ ] I'm ready to contribute to this (and will open a PR soon)
- [ ] I'd like to have a try (with the help of the maintainers)
- [X] No, thanks

**Comment by coder3101** [comment]
Which OS are you using?

**Comment by iSuckBig** [comment]
> Which OS are you using?

<Windows 10 22H2, OS Build: 19045.2846 , and arch 64-bit>

**Comment by worker2011** [comment]
That's true, I hope it can introduce something like a "markerplace" so we can develop relative themes or change the background to ones we want.

**Comment by hiensumi** [comment]
> That's true, I hope it can introduce something like a "markerplace" so we can develop relative themes or change the background to ones we want.

There are only 2 maintainers :)

**Comment by ouuan** [comment]
> There are only 2 maintainers :)

That's not the crucial part. We earn almost nothing (received little donation) and now we no longer do competitive programming ourselves.
